src/call_wrapper.ts

```diff
@@ -3,10 +3,10 @@ import Module = WebAssembly.Module;
 
 class CallWrapper {
 
-    private static fillMemory(parameters: any[], writeableMemory: Uint32Array) {
+    private static fillMemory(parameters: any[], writeableMemory: Uint32Array, initialSize: number) {
```

Comment:
Make method non-static and use the `initialSize` field of the `CallWrapper` instead of passing it as a parameter.